src/components/ArticlePage.jsx

```diff
@@ -31,6 +35,49 @@ export const ArticlePage = ({}) => {
     }
   }
 
+  const handleCommentChange = (event) => setCommentInput(event.target.value);
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    if (!loggedInUser) {
+      setErrorMessage("You must be logged in to post a comment.");
```

Comment:
If you're going to do this, make it clear that a user is logged in. I'd strongly recommend having a default user stored in context.